lib/assignable_values/active_record/restriction/base.rb

```diff
@@ -274,8 +287,13 @@ def ensure_values_given
           @values or @options[:through] or raise NoValuesGiven, 'You must supply the list of assignable values by either a block or :through option'
         end
 
+        def validate_supported_options!
+          unsupported_options = @options.keys - SUPPORTED_OPTIONS
+          if unsupported_options.any?
+            raise UnsupportedOption, "The following options are not supported: #{unsupported_options.join(', ')}"
```

Comment:
In der Fehlermeldung sollten wir noch einen Doppelpunkt vor jede nicht unterstützte Option schreiben.